eth2prysm/values.yaml

```diff
@@ -0,0 +1,30 @@
+# Default values for eth2prysm.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+namespace: prysm
+ethereumNetwork: pyrmont
+imageVersion: v1.3.2
+
+beacon:
+  name: beacon
+  label: beacon
+  serviceAccount: beacon
+  dataMountPath: /data/prysm/beacon
```

Comment:
wondering whether we can always use a static data mount path? `dataHostPath` is up to users configurations and their data dir structure, but `dataMountPath` only matters to the client's container and its `--data-dir` argument . we can add this information in the readme/docs for troubleshooting reference.

Reply:
Removed.